Topic 3/src/main/java/bootcamp/exercises/Roman.java

```diff
@@ -0,0 +1,55 @@
+package bootcamp.exercises;
+
+public class Roman {
+	private final char[] romanNumbers= {' ','I','V','X','L','C','D','M'};
```

Comment:
[High] If you have ' '  as a valid symbol, then you could write something like MCM LIV and it would say that it's valid and the number is 1954.
Remove the '', because you don't need it